documentation/project/contributing/building-and-running-without-docker.md

```diff
@@ -0,0 +1,38 @@
+# Build & Run Without Docker
+
+Once you installed the prerequisites and cloned the repository, run the following steps in your project directory:
+
+## <a name="run_without_docker_quickly"></a> Running Without Docker Quickly
+- [Download the latest kafka-ui jar file](https://github.com/provectus/kafka-ui/releases)
+- Then run the following command.
+```sh
+java -Dspring.config.additional-location=<path-to-application-local.yml> -jar <path-to-kafka-ui-jar>
+```
+- Example of how to configure clusters in the [application-local.yml](https://github.com/provectus/kafka-ui/blob/master/kafka-ui-api/src/main/resources/application-local.yml) configuration file.
+
+## <a name="build_and_run_without_docker"></a> Building And Running Without Docker
+> **_NOTE:_**  If you want to get kafka-ui up and running locally quickly, then just follow [Running Without Docker](#run_without_docker_quickly)
+
+> Skip the maven tests as they might not be successful
+
+> Comment out `com.spotify` plugin in `kafka-ui-api` pom.xml
```

Comment:
that's just a temporal solution and for arm machines

Reply:
Hii I checked this with a centos node and an ubuntu node, both of the nodes didnt have docker installed and it gave an error during the building of kafka-ui-api, and after commenting the spotify docker plugin it built successfully.

Reply:
ahh yeah, in case you don't have docker that's a thing as well, sorry